Tools/tsan/suppressions_free_threading.txt

```diff
@@ -9,6 +9,10 @@
 race:get_allocator_unlocked
 race:set_allocator_unlocked
 
+# https://gist.github.com/mpage/ff1c7094bc8237d98387678d5f52058f
+race_top:_PyThreadState_MustExit
```

Comment:
Why is this a newly reported race?

If possible, I'd prefer we mention some specific unit test to reproduce the race. That makes it easier to remove the suppression later.

Reply:
This is triggered by the unit test that I added. I guess there weren't previously any tests run as part of TSAN that forced daemon threads to exit? I'll add a reference to the test.